src/kernel/xmlreader.c

```diff
@@ -1604,6 +1604,8 @@ static void parse_ai(race * rc, xmlNodePtr node)
         rc->flags |= RCF_MOVERANDOM;
     if (xml_bvalue(node, "learn", false))
         rc->flags |= RCF_LEARN;
+    if (xml_bvalue(node, "moveattack", false))
+        rc->flags |= RCF_ATTACK_MOVED;
```

Comment:
Jein, das wird eigentlich schon durch !r->land || is_guard() erreicht. Das hier bewirkt zusätzlich, dass Seeschlangen PIRATERIE ausführen und dann sofort attackieren können. Andernfalls müssten sie, nachdem sie ihr Opfer erreicht haben, mit der Attacke noch eine Runde warten, weil in monster_attack() monster_is_waiting() abgefragt wird, was wiederrum UFL_MOVED abfragt. Und das würde die PIRATERIE ziemlich sinnlos machen. Deshalb dürfen Monster mit RCF_ATTACK_MOVED direkt nach der Bewegung angreifen. Ich hoffe, das war verständlich.